examples/tts/conf/zh/fastpitch_align_22050.yaml

```diff
@@ -40,10 +40,12 @@ model:
   learn_alignment: true
   bin_loss_warmup_epochs: 100
 
-  n_speakers: 1
+  n_speakers: 1958
```

Comment:
may i know how many speakers exist in the training data? and what is the maximum speaker ID? 

Just wanted to confirm if `n_speakers` is well set up. We suggested to specify `n_speakers = max_value_of_speaker_IDs + 1`

Reply:
I would suggest to create a new config file named as `fastpitch_align_multispeaker_22050.yaml` to differentiate it from a single-speaker config.

Reply:
> may i know how many speakers exist in the training data? and what is the maximum speaker ID?
> 
> Just wanted to confirm if `n_speakers` is well set up. We suggested to specify `n_speakers = max_value_of_speaker_IDs + 1`. But you can always remap the original speaker IDs into the range from 1 to `num_of_speakers`, and assign `n_speakers = num_of_speakers + 1`.

I have sorted all the speaker_id of AISHELL3 and it looks like this:
```
SSB0005
SSB0009
SSB0011
SSB0012
SSB0016
...
SSB1902
SSB1918
SSB1935
SSB1939
SSB1956
```
the largest is `SSB1956` so I use the last 4 digits as speaker_id and set `n_speakers` to 1958.
Do you think we should set `n_speakers` to 1957, or, map them to a continuous range of numbers?

Reply:
> may i know how many speakers exist in the training data? and what is the maximum speaker ID?
> 
> Just wanted to confirm if `n_speakers` is well set up. We suggested to specify `n_speakers = max_value_of_speaker_IDs + 1`. But you can always remap the original speaker IDs into the range from 1 to `num_of_speakers`, and assign `n_speakers = num_of_speakers + 1`.

I have sorted all the speaker_id of AISHELL3 and it looks like this:
```
SSB0005
SSB0009
SSB0011
SSB0012
SSB0016
...
SSB1902
SSB1918
SSB1935
SSB1939
SSB1956
```
the largest is `SSB1956` so I use the last 4 digits as speaker_id and set `n_speakers` to 1958.
Do you think we should set `n_speakers` to 1957, or, map them to a continuous range of numbers?

Reply:
> Another comment related, I would suggest to create a new config file named as `fastpitch_align_multispeaker_22050.yaml` to differentiate it from a single-speaker config.

Added.

Reply:
> the largest is SSB1956 so I use the last 4 digits as speaker_id and set n_speakers to 1958.
Do you think we should set n_speakers to 1957, or, map them to a continuous range of numbers?

It is fine to specify it as 1958, and you need to be careful to specify the speaker id during inference. We highly suggest to map to a range of integers starting from 1, but it is not necessary.

Reply:
> It is fine to specify it as 1958, and you need to be careful to specify the speaker id during inference. We highly suggest to map to a range of integers starting from 1, but it is not necessary.

Thanks! I will add a new PR for remapping the speaker id of AISHELL-3 later.